skunkworks_crow/src/main/java/org/odk/share/tasks/UploadJob.java

```diff
@@ -352,6 +352,9 @@ private void sendBlankForm(Cursor cursor) {
                 sbResult.append(getContext().getString(R.string.id, formId) + " " +
                         getContext().getString(R.string.success, getContext().getString(R.string.blank_form_count,
                                 getContext().getString(R.string.sent))));
+            } else {
```

Comment:
Please undo this change as it is not unrelated

Reply:
@shobhitagarwal1612 Sir, Undid it.